.github/workflows/ci-nightly.yaml

```diff
@@ -0,0 +1,20 @@
+name: Kata Containers Nightly CI
+on:
+  schedule:
+    cron: '0 0 * * *'
+
+env:
+  COMMIT_HASH: ${GITHUB_REF}
+
+jobs:
+  set-fake-pr-number:
+    runs-on: ubuntu-latest
+    run: |
+      echo ::set-env name=PR_NUMBER::$(date +%Y%m%d%H%M%S)
```

Comment:
IIRC - I think that I saw `set-env` being deprecated and replaced with using the environment variable file? https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
Fixed, thanks for the pointer!